views/question.hbs

```diff
@@ -28,33 +28,63 @@
 
       <div class="row question-content">
         <div class="col-sm-1 text-center">
-          <form action="/question/{{question.id}}/up" method="post">
-            <input type="hidden" name="_method" value="PATCH">
-            <button type="submit" class="not-button" onclick="questionUpCookie()" id="button-up"><i class="glyphicon glyphicon-triangle-top not-voted" id="up"></i></button>
-          </form>
-          <span class="nbOfVotes">{{question.score}}</span><br />
-          <form action="/question/{{question.id}}/down" method="post">
-            <input type="hidden" name="_method" value="PATCH">
-            <button type="submit" class="not-button" onclick="questionDownCookie()" id="button-down"><i class="glyphicon glyphicon-triangle-bottom not-voted" id="down"></i></button>
-          </form>
+          {{#if user}}
+            {{#each @root.question.voteHistory}}
+              {{#eq user @root.user.username}}
+                {{#eq current 1}}
+                  <button class="not-button"  id="button-up"><i class="glyphicon glyphicon-triangle-top up-voted" id="up"></i></button><br />
+                  <span class="nbOfVotes">{{@root.question.score}}</span><br />
+                  <form action="/question/{{@root.question.id}}/downupped" method="post">
+                    <input type="hidden" name="_method" value="PATCH">
+                    <button type="submit" class="not-button" id="button-down"><i class="glyphicon glyphicon-triangle-bottom not-voted" id="down"></i></button>
```

Comment:
Why is the `i` element being used here? also, the closing tag of `button` should be on its own line, as should the element enclosed within `button`.